### ISSUE TITLE
ERROR: Could not build wheels for rapidfuzz, which is required to install pyproject.toml-based projects

### ISSUE THREAD

**Issue by Biaocsu** [maxbachmann/rapidfuzz#159]
```
Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple, http://mirrors.aliyun.com/pypi/simple, http://pypi.douban.com/sim
ple, http://pypi.mirrors.ustc.edu.cn/simple
Collecting rapidfuzz
  Using cached https://mirrors.bfsu.edu.cn/pypi/web/packages/21/c5/92864654ef66a451fd9d014213f56d9e9b62bde7415f6962e2ea51ae6d07
/rapidfuzz-1.8.2.tar.gz (526 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: rapidfuzz
  Building wheel for rapidfuzz (pyproject.toml) ... error
  ERROR: Command errored out with exit status 1:
   command: 'C:\users\liubiao\anaconda3\python.exe' 'C:\users\liubiao\anaconda3\lib\site-packages\pip\_vendor\pep517\in_process
\_in_process.py' build_wheel 'C:\Users\liubiao\AppData\Local\Temp\tmphdreedmu'
       cwd: C:\Users\liubiao\AppData\Local\Temp\pip-install-32_w9_xl\rapidfuzz_40c5069bda7a47bf8dc2e29320b156d3
  Complete output (605 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.6
  creating build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\fuzz.py -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\levenshtein.py -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\process.py -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\string_metric.py -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\utils.py -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\__init__.py -> build\lib.win-amd64-3.6\rapidfuzz
  running egg_info
  writing src\rapidfuzz.egg-info\PKG-INFO
  writing dependency_links to src\rapidfuzz.egg-info\dependency_links.txt
  writing requirements to src\rapidfuzz.egg-info\requires.txt
  writing top-level names to src\rapidfuzz.egg-info\top_level.txt
  reading manifest file 'src\rapidfuzz.egg-info\SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no files found matching '*.txx' under directory 'src\rapidfuzz-cpp\rapidfuzz'
  adding license file 'LICENSE'
  writing manifest file 'src\rapidfuzz.egg-info\SOURCES.txt'
  copying src\rapidfuzz\__init__.pyi -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\fuzz.pyi -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\process.pyi -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\py.typed -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\string_metrics.pyi -> build\lib.win-amd64-3.6\rapidfuzz
  copying src\rapidfuzz\utils.pyi -> build\lib.win-amd64-3.6\rapidfuzz
  running build_ext
  building 'rapidfuzz.cpp_process' extension
  creating build\temp.win-amd64-3.6
  creating build\temp.win-amd64-3.6\Release
  creating build\temp.win-amd64-3.6\Release\src
  creating build\temp.win-amd64-3.6\Release\src\rapidfuzz-cpp
  creating build\temp.win-amd64-3.6\Release\src\rapidfuzz-cpp\rapidfuzz
  creating build\temp.win-amd64-3.6\Release\src\rapidfuzz-cpp\rapidfuzz\details
  C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/rapidfu
zz-cpp/ -IC:\users\liubiao\anaconda3\include -IC:\users\liubiao\anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual St
udio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" "-IC:\Program Files (x86)\Windows Ki
ts\8.1\include\shared" "-IC:\Program Files (x86)\Windows Kits\8.1\include\um" "-IC:\Program Files (x86)\Windows Kits\8.1\includ
e\winrt" /EHsc /Tpsrc/cpp_process.cpp /Fobuild\temp.win-amd64-3.6\Release\src/cpp_process.obj /EHsc /std:c++14 /O2 /W4 /DNDEBUG
 /DVERSION_INFO=\"1.8.2\"
  cl : Command line warning D9025 : overriding '/W3' with '/W4'
  cpp_process.cpp
  src/rapidfuzz-cpp/rapidfuzz/string_metric.hpp: warning C4819: The file contains a character that cannot be represented in the
 current code page (936). Save the file in Unicode format to prevent data loss
  c:\users\liubiao\appdata\local\temp\pip-install-32_w9_xl\rapidfuzz_40c5069bda7a47bf8dc2e29320b156d3\src\cpp_common.hpp(57): w
arning C4800: 'uint8_t': forcing value to bool 'true' or 'false' (performance warning)
  src/cpp_process.cpp(1502): warning C4127: conditional expression is constant
  src/cpp_process.cpp(6262): warning C4127: conditional expression is constant
  src/cpp_process.cpp(6270): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(9076): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(9113): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(9127): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(10009): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(10021): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(10368): warning C4127: conditional expression is constant
  src/cpp_process_cdist.cpp(10380): warning C4127: conditional expression is constant
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2
  ----------------------------------------
  ERROR: Failed building wheel for rapidfuzz
Failed to build rapidfuzz
ERROR: Could not build wheels for rapidfuzz, which is required to install pyproject.toml-based projects
```

**Comment by maxbachmann** [comment]
Thanks for reporting. Apparently I forgot to build wheels for Python3.6 on MacOs and WIndows: https://github.com/maxbachmann/RapidFuzz/commit/df568cafbab56359480f0d20b14f8a01b7fef137

If the build is successful, I will release a new version 1.8.3 later today which includes those wheels.

**Comment by maxbachmann** [comment]
@Biaocsu This should be fixed with v1.8.3

**Comment by Biaocsu** [comment]
@maxbachmann thanks, I saw it
